source/optional_features/prefill_user_details/index.html.md.erb

```diff
@@ -29,9 +29,9 @@ For example:
  }
 ```
 
-You can leave out any of the parameters.
+All of the parameters in `billing_address` are optional.
 
-The `country` parameter must be a valid <a href="https://www.iso.org/iso-3166-country-codes.html" target="blank">ISO 3166 code</a>. If the parameter is invalid or missing, the __Country or territory__ field will default to 'GB'.
+If you include the `country` parameter, it must be a valid <a href="https://www.iso.org/iso-3166-country-codes.html" target="blank">ISO 3166 code</a>. If the parameter is invalid or missing, the __Country or territory__ field will default to 'Great Britain'.
```

Comment:
It actually defaults to country code 'GB' but the user is going to see 'United Kingdom'

Reply:
Ah thanks for clarifying @SandorArpa!